dist/Passable.min.js

```diff
@@ -1 +1 @@
{return e&&e.__esModule?e:{default:e}}Object.defineProperty(t,"__esModule",{value:!0});var o=r(33),u=n(o),i=r(34),s=n(i),a=r(35),l=n(a);t.default={allOf:u.default,anyOf:s.default,noneOf:l.default},e.exports=t.default},function(e,t,r){"use strict";function n(e,t,r){var n=Object.keys(t);return 0!==n.length&&n.every(function(n){return!0===(0,u.default)(e,n,t,r)})}Object.defineProperty(t,"__esModule",{value:!0}),t.default=n;var o=r(3),u=function(e){return e&&e.__esModule?e:{default:e}}(o);e.exports=t.default},function(e,t,r){"use strict";function n(e,t,r){return Object.keys(t).some(function(n){return!0===(0,u.default)(e,n,t,r)})}Object.defineProperty(t,"__esModule",{value:!0}),t.default=n;var o=r(3),u=function(e){return e&&e.__esModule?e:{default:e}}(o);e.exports=t.default},function(e,t,r){"use strict";function n(e,t,r){var n=Object.keys(t);return 0!==n.length&&n.every(function(n){return!0!==(0,u.default)(e,n,t,r)})}Object.defineProperty(t,"__esModule",{value:!0}),t.default=n;var o=r(3),u=function(e){return e&&e.__esModule?e:{default:e}}(o);e.exports=t.default},function(e,t,r){"use strict";function n(e){var t=null;if("function"!=typeof e)return!1;try{var r=e();t=void 0!==r&&null!==r&&r.hasOwnProperty("valid")?r.valid:"boolean"!=typeof r||(r||!1)}catch(e){t=!1}return!!t}Object.defineProperty(t,"__esModule",{value:!0}),t.default=n,e.exports=t.default},function(e,t){e.exports={name:"passable",version:"5.7.8",description:"Isomorphic Data Model Validations",main:"./dist/Passable.min.js",author:"Evyatar <evyatar.a@fiverr.com>",license:"MIT",scripts:{build:"cross-env WEBPACK_ENV=build webpack","build:watch":"webpack --watch",dev:"cross-env WEBPACK_ENV=dev webpack --progress --colors --watch","test:custom":"mocha --compilers js:babel-core/register --colors -w './src/**/*custom.spec.js'","test:global_custom":"mocha --compilers js:babel-core/register --colors -w './src/spec/passable.api.global-custom.spec.js'","test:enforce":"mocha --compilers js:babel-core/register --colors -w './src/enforce/spec.js'","test:pass_runner":"mocha --compilers js:babel-core/register --colors -w './src/pass_runner/spec.js'","test:watch":"mocha --compilers js:babel-core/register --colors -w './src/**/*spec.js'",test:"mocha --compilers js:babel-core/register --colors './src/**/*spec.js'",eslint:"eslint -c .eslintrc './src/**/*.js'; exit 0"},repository:{type:"git",url:"git+https://github.com/fiverr/passable.git"},bugs:{url:"https://github.com/fiverr/passable/issues"},homepage:"https://github.com/fiverr/passable#readme",devDependencies:{"babel-cli":"^6.24.1","babel-core":"^6.25.0","babel-eslint":"^8.0.0","babel-loader":"^7.1.0","babel-plugin-add-module-exports":"^0.2.1","babel-plugin-transform-class-properties":"^6.24.1","babel-plugin-webpack-aliases":"^1.0.3","babel-preset-env":"^1.6.0","babel-preset-flow":"^6.23.0",chai:"^4.1.1","cross-env":"^5.1.1",eslint:"^4.5.0","eslint-loader":"^1.9.0","eslint-plugin-flowtype":"^2.35.0","flow-bin":"^0.57.3",mocha:"^4.0.1",webpack:"^3.5.5"}}}])});
```

Comment:
any reason why we commit `dist` folder rather than having it as part of automation / publication process?

Reply:
I think it's for manual downloads (a-la vendors). @ealush ?

Reply:
As @omrilotan noted, so users can download the script file directly from github. Passable can be embedded directly as a script file.